test/markup/python-repl/sample.expect.txt

```diff
@@ -11,7 +11,7 @@ foo = 42"
 <span class="hljs-meta">&gt;&gt;&gt;</span> <span class="python"><span class="hljs-string">"""</span></span>
 <span class="hljs-meta">...</span> <span class="python"><span class="hljs-string">abc</span></span>
 <span class="hljs-meta">...</span> <span class="python"><span class="hljs-string">"""</span></span>
-<span class="hljs-meta">&gt;&gt;&gt;</span> <span class="python"><span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">test</span><span class="hljs-params">()</span>:</span></span>
+<span class="hljs-meta">&gt;&gt;&gt;</span> <span class="python"><span class="hljs-function"><span class="hljs-keyword">def</span> <span class="hljs-title">test</span>(<span class="hljs-params"></span>):</span></span>
```

Comment:
I'll accept as is, but would it be too much trouble to handle the edge case of () so it doesn't produce noise?

It'd prolly be something like:

```
  var PARAMS = {
    className: 'params',
    variants: [
     {begin: /\(/, end: /\)/, excludeBegin: true, excludeEnd: true, contains: ['self', PROMPT, NUMBER, STRING, hljs.HASH_COMMENT_MODE]},
     {begin: /\(\s*\)/, skip: true, className: null }
    ]
    
  };
```

Or some such.

Reply:
Ok, it's included.